Packs/Core/Playbooks/playbook-Msiexec_execution_of_an_executable_from_an_uncommon_remote_location.yml

```diff
@@ -0,0 +1,1265 @@
+      '#none#':
+      - "8"
+    scriptarguments:
+      command_line:
+        complex:
+          root: alert
+          accessor: cgocmd
+          transformers:
+          - operator: splitAndTrim
+            args:
+              delimiter:
+                value:
+                  simple: |2+
+
+          - operator: RemoveEmpty
+            args:
+              empty_values: {}
+              remove_keys:
+                value:
+                  simple: "true"
+          - operator: join
+            args:
+              separator: {}
+      custom_patterns:
+        simple: ((cmd|type)<?\s*\w+\.\w{1,3}(\|cmd)?(\:\w+)?)
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": 1030
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "32":
+    id: "32"
+    taskid: 6d81614b-8e09-44c7-8a10-4f9e1bd02f8d
+    type: title
+    task:
+      id: 6d81614b-8e09-44c7-8a10-4f9e1bd02f8d
+      version: -1
+      name: Early Containment
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "33"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -120,
+          "y": -90
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "33":
+    id: "33"
+    taskid: 567ba8e5-5ded-41ab-839e-0540e4f66b6e
+    type: regular
+    task:
+      id: 567ba8e5-5ded-41ab-839e-0540e4f66b6e
+      version: -1
+      name: Terminate Causality (CGO)
+      description: Terminate a process tree by its causality ID. Available only for Cortex XSIAM 2.4.
+      script: '|||core-terminate-causality'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "35"
+    scriptarguments:
+      agent_id:
+        simple: ${alert.agentid}
+      causality_id:
+        simple: ${alert.cid}
+      timeout_in_seconds:
+        simple: "180"
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -120,
+          "y": 1555
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "35":
+    id: "35"
+    taskid: 7cd09d57-8e0e-4070-8873-f5f6c8c9be3c
+    type: collection
+    task:
+      id: 7cd09d57-8e0e-4070-8873-f5f6c8c9be3c
+      version: -1
+      name: Approve the endpoint isolation & Block URL
+      description: |-
+        Endpoint Isolation is recommended since the following verdicts have been confirmed:
```

Comment:
the URL wasnt necessarily malicious. there are also other factors for the isolation (your investigation phase)